### ISSUE TITLE
Trainer Progressbar Not Showing the Metrics in Kaggle notebooks

### ISSUE THREAD

**Issue by aribornstein** [PyTorchLightning/pytorch-lightning#7278]
## 🐛 Bug

When training the a multi label flash image classification model I'm seeing the following output on kaggle

![image](https://user-images.githubusercontent.com/3045407/116565828-df8b0b80-a90e-11eb-9f41-727d3323257c.png)

### To Reproduce

1. Create Kaggle Notebook for ranzcr challenge
2. Set accelerator to gpu
3. Install Flash !python -m pip install git+https://github.com/PyTorchLightning/lightning-flash
3. Run the following code

```python
import os.path as osp
from typing import List, Tuple

import pandas as pd
from torchmetrics import F1

import flash
from flash.core.classification import Labels
from flash.vision import ImageClassificationData, ImageClassifier
from flash.vision.classification.data import ImageClassificationPreprocess

df = pd.read_csv("../input/ranzcr-clip-catheter-line-classification/train.csv")
label_classes = ['ETT - Abnormal', 'ETT - Borderline','ETT - Normal', 'NGT - Abnormal', 'NGT - Borderline',
                 'NGT - Incompletely Imaged', 'NGT - Normal', 'CVC - Abnormal',
                 'CVC - Borderline', 'CVC - Normal', 'Swan Ganz Catheter Present']

train_filepaths = [f"../input/ranzcr-clip-catheter-line-classification/train/{i}.jpg" for i in df['StudyInstanceUID']]
train_labels = df[label_classes].values.tolist()

datamodule = ImageClassificationData.from_filepaths(
    train_filepaths=train_filepaths,
    train_labels=train_labels,
    preprocess=ImageClassificationPreprocess(image_size=(128, 128)),
    val_split=0.1,  # Use 10 % of the train dataset to generate validation one.
)

# 3. Build the model
model = ImageClassifier(
    backbone="resnet18",
    num_classes=len(label_classes),
    multi_label=True,
    metrics=F1(num_classes=len(label_classes)),
)

# 4. Create the trainer. Train on 1 gpu for 10 epochs.
trainer = flash.Trainer(gpus=1, max_epochs=10)

# 5. Train the model
trainer.finetune(model, datamodule=datamodule, strategy="freeze")
```

### Expected behavior

To see the metrics and as the trainer trains. 

### Environment
Kaggle GPU accelerator 
**Note**: PM me for Notebook access

**Comment by tchaton** [comment]
https://github.com/tqdm/tqdm/issues/1165
